docs/admin/security/connection-logs.md

```diff
@@ -0,0 +1,103 @@
+# Connection Logs
+
+The Connection Log page in the dashboard allows **Auditors** to monitor workspace agent connections.
+
+## Workspace app connections
+
+The connection log contains a complete record of all **workspace app** connections.
+These originate from within the Coder deployment, and thus the connection log
+is a source of truth for these events.
+
+## Browser port forwarding
+
+The connection log contains a complete record of all workspace port forwarding
+performed via the web dashboard.
+
+## SSH & IDE sessions
+
+The connection log aims to capture a record of SSH & IDE sessions to workspaces.
+These events are reported by workspace agents, and their receipt by the server
+is not guaranteed.
+
+## Filtering logs
+
+Connection logs can be filtered with the following parameters:
+
+- `organization` - The name or ID of the organization of the workspace being
+     connected to.
+- `workspace_owner` - The username of the owner of the workspace being connected
+    to.
+- `type` - The type of the connection (i.e. SSH, VSCode, Workspace App).
+    An exhaustive list is
+    [available here](https://pkg.go.dev/github.com/coder/coder/v2/codersdk#ConnectionType).
```

Comment:
```suggestion
    For more connection types, consult the
    [codersdk package](https://pkg.go.dev/github.com/coder/coder/v2/codersdk#ConnectionType)
    page in the Go database.
```

I'm not 100% on this one's wording, but basically the link text should be descriptive